kattelmod/systems/mkat/session.py

```diff
@@ -69,5 +78,8 @@ async def capture_done(self) -> None:
             await self.sdp.capture_done()
 
     async def product_deconfigure(self) -> None:
+        if self._telstate:
```

Comment:
Maybe do this as the final step? Just thinking of how site works with telstate the one to turn off the lights.

Reply:
I was thinking of this as disconnecting from the service before it disappears from under us, just in case doing it the other way around causes any warnings about losing connection to the server.

Reply:
Good point.